src/App.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
 
+var todoList = [
+  {
+    id: 1,
+    title: "Wake up"
+  },
+  {
+    id: 2,
+    title: "Take a shower"
+  },
+  {
+    id: 3,
+    title: "Walk the dog"
+  }
+];
+
 function App() {
```

Comment:
Don't forget to format your code. It's going to make reading code a lot easier especially as it grows. Your IDE should have a friendly 'hotkey' or in settings a command. It's possible you might need to download an extension to provide formatting if it doesn't come out of the box.